### PR TITLE
Add header link to documentation

### PR DESCRIPTION
**Pull request by aaronrothschild**
Added a link to the Documentation at the TOP of the settings page instead of at the bottom where users may miss it.

### CODE REVIEW

**Review by mickmister — APPROVED** [review]
:+1:

**Comment by hanzei** [comment]
@justinegeffen Could you please take a look at this PR?

File: plugin.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
         "bundle_path": "webapp/dist/main.js"
     },
     "settings_schema": {
+         "header": "To setup this plugin you must create a 'Zoom App' with a Zoom Administrator's account - See [documentation here](https://mattermost.gitbook.io/plugin-zoom/installation/zoom-configuration)",
```
Comment (justinegeffen) [inline plugin.json:22]:
```suggestion
         "header": "To set up this plugin you first need to create a Zoom App using a Zoom Administrator account. Visit the [documentation for configuration steps](https://mattermost.gitbook.io/plugin-zoom/installation/zoom-configuration).",
```

**Comment by justinegeffen** [comment]
Thanks, @hanzei.

@aaronrothschild, I made a suggestion but I'm not sure whether it changes the way the UI is displayed if at all. LMK if there are fundamental changes.

**Comment by aaronrothschild** [comment]
I can't review this for some reason LGTM.

**Pull request merged**